src/runtime-rs/crates/hypervisor/src/qemu/inner.rs

```diff
@@ -57,28 +55,72 @@ impl QemuInner {
         let vm_path = [KATA_PATH, self.id.as_str()].join("/");
         std::fs::create_dir_all(vm_path)?;
 
-        let mut command = std::process::Command::new(&self.config.path);
+        let mut cmdline = QemuCmdLine::new(&self.id, &self.config)?;
+
+        // If there's a Vsock Device in `self.devices` the vhost-vsock file
+        // descriptor needs to stay open until the qemu process launches.
+        // This is why we need to store it in a variable at this scope.
+        let mut _vhost_fd = None;
```

Comment:
Since the only reason for this variable to exist is to close the fd, the `vhost` wording is irrelevant. Also, as discussed on slack, we already know that there will be other type of file descriptors that should be handled the same.

What about giving a generic name to the variable, e.g. `passed_fd` or anything else meaningful ?